.github/workflows/main.yml

```diff
@@ -210,6 +210,8 @@ jobs:
           registry-url: https://registry.npmjs.org
 
       - name: Publish to NPM
-        run: npm publish
+        run: |
+          TAG=$(echo $GITHUB_REF_NAME | grep -oP '^v\d+\.\d+\.\d+-?\K(\w+)?')
```

Comment:
Does this work on Windows too? I mean that `grep` is not installed per default

Reply:
It doesn't matter, this script is specifically for running on Ubuntu within GitHub Actions.

Reply:
Ah ok, so `npm publish` is not called manually? Is it triggered over a Github Release?

Reply:
It's triggered when we create and push a `v*` git tag (@jonkoops although maybe we should make the check more conservative, or create another workflow so that it could be triggered by a new GitHub Release).

Reply:
Instead of using `$GITHUB_REF_NAME` you could also get the version using `npm view version leaflet` (which will not include the `v` prefix). But this works excellently as well.